cms/static/js/views/active_video_upload_list.js

```diff
@@ -10,6 +11,7 @@ define(
                 "dragleave .file-drop-area": "dragleave",
                 "drop .file-drop-area": "dragleave"
             },
+            pageUnloadMessage: pageUnloadMessage,
```

Comment:
Ah cool, that makes sense.  But given your point below about the tests needing to check the exact message, I've removed both the unnecessary var and the view.pageUnloadMessage.